repos/system_upgrade/el7toel8/workflows/inplace_upgrade.py

```diff
@@ -50,15 +50,15 @@ class ReportsPhase(Phase):
 
     # The following two phases have been removed but are kept here for reference or in case they are needed again
 
-    #class AttachPackageReposPhase(Phase):
+    # class AttachPackageReposPhase(Phase):
```

Comment:
Open question: Are we going need these 2 classes in the nearest future? 
If not I'd suggest to remove them, git knows what we did last summer.

Reply:
We kept them here for reference, even if we can ask git about what we did last summer. Any bigger refactoring makes git scream and run around like a headless chicken with arms waving all around forcing us to ask them multiple times...

Anyway I know where you are going, but we might want to remove it. Just not with this PR.

Reply:
Closing this in favor of separate PR.